### PR TITLE
Adding the Mic pulse icons

### PR DESCRIPTION
**Pull request by sfilipi**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/6dfe3b17a1a07660a7071df9747b6b0a3e601a69'>6dfe3b17a1a07660a7071df9747b6b0a3e601a69</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
DividerMinimalPerf.default | [301](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DividerMinimalPerf.default.html) | [277](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.09:1
AttachmentMinimalPerf.default | [132](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AttachmentMinimalPerf.default.html) | [125](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.06:1
PortalMinimalPerf.default | [146](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/PortalMinimalPerf.default.html) | [138](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.06:1
AccordionMinimalPerf.default | [119](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AccordionMinimalPerf.default.html) | [114](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 1.04:1
ImageMinimalPerf.default | [312](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ImageMinimalPerf.default.html) | [301](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1.04:1
LayoutMinimalPerf.default | [298](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/LayoutMinimalPerf.default.html) | [287](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.04:1
TableMinimalPerf.default | [344](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TableMinimalPerf.default.html) | [330](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.04:1
ButtonMinimalPerf.default | [143](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ButtonMinimalPerf.default.html) | [139](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.03:1
ButtonSlotsPerf.default | [465](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ButtonSlotsPerf.default.html) | [453](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.03:1
GridMinimalPerf.default | [287](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/GridMinimalPerf.default.html) | [278](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.03:1
ListCommonPerf.default | [547](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ListCommonPerf.default.html) | [532](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ListCommonPerf.default_base.html) | 1.03:1
PopupMinimalPerf.default | [534](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/PopupMinimalPerf.default.html) | [518](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.03:1
ReactionMinimalPerf.default | [323](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ReactionMinimalPerf.default.html) | [313](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.03:1
TextMinimalPerf.default | [299](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TextMinimalPerf.default.html) | [289](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.03:1
FormMinimalPerf.default | [338](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/FormMinimalPerf.default.html) | [333](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.02:1
HeaderMinimalPerf.default | [298](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/HeaderMinimalPerf.default.html) | [291](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.02:1
HeaderSlotsPerf.default | [635](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/HeaderSlotsPerf.default.html) | [620](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.02:1
TreeWith60ListItems.default | [150](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TreeWith60ListItems.default.html) | [147](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.02:1
AlertMinimalPerf.default | [224](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AlertMinimalPerf.default.html) | [222](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1.01:1
AnimationMinimalPerf.default | [461](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AnimationMinimalPerf.default.html) | [456](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.01:1
AttachmentSlotsPerf.default | [919](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AttachmentSlotsPerf.default.html) | [908](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.01:1
AvatarMinimalPerf.default | [157](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AvatarMinimalPerf.default.html) | [155](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.01:1
ButtonOverridesMissPerf.default | [1267](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1253](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1.01:1
ChatMinimalPerf.default | [620](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ChatMinimalPerf.default.html) | [616](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.01:1
DialogMinimalPerf.default | [646](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DialogMinimalPerf.default.html) | [641](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.01:1
DropdownManyItemsPerf.default | [566](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DropdownManyItemsPerf.default.html) | [559](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.01:1
FlexMinimalPerf.default | [238](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/FlexMinimalPerf.default.html) | [236](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.01:1
InputMinimalPerf.default | [1093](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/InputMinimalPerf.default.html) | [1084](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.01:1
ItemLayoutMinimalPerf.default | [987](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [979](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.01:1
ListMinimalPerf.default | [430](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ListMinimalPerf.default.html) | [424](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.01:1
ProviderMergeThemesPerf.default | [1069](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1060](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.01:1
ProviderMinimalPerf.default | [344](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ProviderMinimalPerf.default.html) | [342](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.01:1
SkeletonMinimalPerf.default | [289](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/SkeletonMinimalPerf.default.html) | [287](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.01:1
TooltipMinimalPerf.default | [901](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TooltipMinimalPerf.default.html) | [895](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.01:1
TreeMinimalPerf.default | [686](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TreeMinimalPerf.default.html) | [679](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.01:1
BoxMinimalPerf.default | [281](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/BoxMinimalPerf.default.html) | [281](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1:1
CarouselMinimalPerf.default | [396](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/CarouselMinimalPerf.default.html) | [395](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1:1
ChatDuplicateMessagesPerf.default | [249](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [248](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1:1
CheckboxMinimalPerf.default | [2273](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/CheckboxMinimalPerf.default.html) | [2271](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1:1
DatepickerMinimalPerf.default | [4845](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DatepickerMinimalPerf.default.html) | [4826](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1:1
DropdownMinimalPerf.default | [2567](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DropdownMinimalPerf.default.html) | [2566](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1:1
EmbedMinimalPerf.default | [3449](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/EmbedMinimalPerf.default.html) | [3445](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1:1
LabelMinimalPerf.default | [320](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/LabelMinimalPerf.default.html) | [319](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1:1
ListWith60ListItems.default | [549](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ListWith60ListItems.default.html) | [548](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ListWith60ListItems.default_base.html) | 1:1
SegmentMinimalPerf.default | [284](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/SegmentMinimalPerf.default.html) | [285](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1:1
SplitButtonMinimalPerf.default | [3692](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [3698](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1:1
TableManyItemsPerf.default | [1614](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TableManyItemsPerf.default.html) | [1613](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1:1
CustomToolbarPrototype.default | [2291](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/CustomToolbarPrototype.default.html) | [2298](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1:1
CardMinimalPerf.default | [464](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/CardMinimalPerf.default.html) | [471](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.99:1
ChatWithPopoverPerf.default | [313](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ChatWithPopoverPerf.default.html) | [317](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.99:1
ListNestedPerf.default | [462](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ListNestedPerf.default.html) | [467](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ListNestedPerf.default_base.html) | 0.99:1
MenuMinimalPerf.default | [716](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/MenuMinimalPerf.default.html) | [721](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/MenuMinimalPerf.default_base.html) | 0.99:1
MenuButtonMinimalPerf.default | [1410](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1429](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 0.99:1
RosterPerf.default | [924](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/RosterPerf.default.html) | [938](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/RosterPerf.default_base.html) | 0.99:1
RadioGroupMinimalPerf.default | [369](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [373](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 0.99:1
SliderMinimalPerf.default | [1407](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/SliderMinimalPerf.default.html) | [1419](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/SliderMinimalPerf.default_base.html) | 0.99:1
TextAreaMinimalPerf.default | [413](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TextAreaMinimalPerf.default.html) | [416](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 0.99:1
ToolbarMinimalPerf.default | [798](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ToolbarMinimalPerf.default.html) | [808](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 0.99:1
VideoMinimalPerf.default | [536](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/VideoMinimalPerf.default.html) | [541](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/VideoMinimalPerf.default_base.html) | 0.99:1
LoaderMinimalPerf.default | [557](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/LoaderMinimalPerf.default.html) | [571](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 0.98:1
StatusMinimalPerf.default | [559](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/StatusMinimalPerf.default.html) | [569](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/StatusMinimalPerf.default_base.html) | 0.98:1
RefMinimalPerf.default | [188](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/RefMinimalPerf.default.html) | [193](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/RefMinimalPerf.default_base.html) | 0.97:1
IconMinimalPerf.default | [491](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/IconMinimalPerf.default.html) | [513](https://fluentuipr.z22.web.core.windows.net/pull/22944/perf-test-northstar/IconMinimalPerf.default_base.html) | 0.96:1

</details><div id="perfComment9424"/>

File: packages/fluentui/CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Documentation
 - Use https for Embed and Video examples @Hirse ([#22738](https://github.com/microsoft/fluentui/pull/22738))
 
+### Features
```
Comment (sfilipi) [inline packages/fluentui/CHANGELOG.md:40]: @GeoffCoxMSFT is this the right spot to add to Changelog

Comment (jurokapsiar) [reply packages/fluentui/CHANGELOG.md]: yes, but moved the features higher in the changelog.

Comment (sfilipi) [reply packages/fluentui/CHANGELOG.md]: Thank you!
Is this good to be merged then? I did test locally.

**Review by jurokapsiar — APPROVED** [review]

**Pull request merged**